.gitignore

```diff
@@ -0,0 +1,30 @@
+# Created by .ignore support plugin (hsz.mobi)
```

Comment:
I prefer to have no .gitignore file in my projects. I rather add a .gitignore with ".gitignore" in the first line.